src/Microsoft.OData.Core/ODataBatchReader.cs

```diff
@@ -16,27 +18,26 @@ namespace Microsoft.OData.Core
 #if ODATALIB_ASYNC
     using System.Threading.Tasks;
 #endif
+
+    using Microsoft.OData.Core.JsonLight;

```

Comment:
The JsonLight namespace should not be used by this base class. It  looks like it's not actually used and this line should be able to safely be deleted.